src/Application.java

```diff
@@ -0,0 +1,58 @@
+import sun.java2d.loops.TransformBlit;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+public class Application {
+
+    public void start() {
+        Transaction[] transactions = getData();
+        Transaction[] orderedTransactions = orderTransactions(transactions);
+        for (Transaction transaction : orderedTransactions) {
+            print(transaction);
+        }
+    }
+
+    public Transaction[] getData() {
+        return new TransactionData().getTransactionData();
+    }
+
+    public Transaction[] orderTransactions(Transaction[] unorderedTransactions) {
+        List<Transaction> transactionList = Arrays.asList(unorderedTransactions);
+        transactionList.sort(new Comparator<Transaction>() {
```

Comment:
This whole `sort` thing is really a one liner. If you've got time try letting IntelliJ guide you with an `alt+enter` or two ;)